src/openforms/registrations/contrib/objects_api/tests/test_backend.py

```diff
@@ -0,0 +1,326 @@
+from datetime import date
+
+from django.test import TestCase
+
+import requests_mock
+from zds_client.oas import schema_fetcher
+from zgw_consumers.test import generate_oas_component
+from zgw_consumers.test.schema_mock import mock_service_oas_get
+
+from openforms.registrations.constants import RegistrationAttribute
+from openforms.registrations.contrib.objects_api.plugin import ObjectsAPIRegistration
+from openforms.submissions.tests.factories import SubmissionFactory
+
+from ...zgw_apis.tests.factories import ZgwConfigFactory
+from ..models import ObjectsAPIConfig
+from .factories import ObjectsAPIConfigFactory
+
+
+@requests_mock.Mocker()
+class ObjectsAPIBackendTests(TestCase):
+    maxDiff = None
+
+    @classmethod
+    def setUpTestData(cls):
+        ObjectsAPIConfigFactory.create(
+            objects_service__api_root="https://objecten.nl/api/v1/",
+            objecttype="https://objecttypen.nl/api/v1/objecttypes/1",
+            objecttype_version=1,
+            productaanvraag_type="terugbelnotitie",
+        )
+        ZgwConfigFactory.create(
+            zrc_service__api_root="https://zaken.nl/api/v1/",
+            drc_service__api_root="https://documenten.nl/api/v1/",
+            ztc_service__api_root="https://catalogi.nl/api/v1/",
+            informatieobjecttype="https://catalogi.nl/api/v1/informatieobjecttypen/1",
+            organisatie_rsin="000000000",
+        )
+
+    def tearDown(self):
+        super().tearDown()
+
+        # Reset the schema fetcher cache, to verify that
+        # the OAS is being retrieved on every submission
+        schema_fetcher.cache = {}
```

Comment:
ZGW consumers installs its django based cache system: https://github.com/maykinmedia/zgw-consumers/blob/master/zgw_consumers/cache.py

I'd move this ordeal to the `setUp` method:

```
def setUp(self):
    super().setUp()

    # ensure the schema cache is cleared before and after each test
    schema_fetcher.cache.clear()
    self.addCleanup(schema_fetcher.cache.clear)
```